core/arch/arm/mm/core_mmu_v7.c

```diff
@@ -733,13 +733,9 @@ static void map_page_memarea_in_pgdirs(const struct tee_mmap_region *mm,
 	attr = mattr_to_desc(1, mm->attr | TEE_MATTR_TABLE);
 	pa = map_page_memarea(mm, ttb[idx]);
 
-	n = ROUNDUP(mm->size, SECTION_SIZE) >> SECTION_SHIFT;
-	while (n--) {
-		assert(!attr || !ttb[idx] || ttb[idx] == (pa | attr));
-		ttb[idx] = pa | attr;
-		idx++;
-		pa += SECTION_SIZE;
-	}
+	assert(!attr || !ttb[idx] || ttb[idx] == (pa | attr));
```

Comment:
Update this, `attr` is never null: `assert(!ttb[idx] || ttb[idx] == (pa | attr));`